Catalyst.Fabric.Authorization.Client/AuthorizationClient.cs

```diff
@@ -3,38 +3,90 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
+using System.Threading;
 using System.Threading.Tasks;
 using Catalyst.Fabric.Authorization.Client.Extensions;
 using Catalyst.Fabric.Authorization.Client.Routes;
 using Catalyst.Fabric.Authorization.Models;
 using Catalyst.Fabric.Authorization.Models.Requests;
 using Newtonsoft.Json;
+using Polly;
 
 namespace Catalyst.Fabric.Authorization.Client
 {
     public class AuthorizationClient
     {
-        
+        private Policy RetryPolicy;
+        private Policy CircuitBreakerPolicy;

```

Comment:
Since this is a local policy, then the circuit will only break when this instance of the policy hits its limit, is that what we want? To be most effective, it would require the client of this library only instantiating one AuthorizationClient.